internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```diff
@@ -0,0 +1,53 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>splunk-otel-collector</id>
+    <version>0.0.0</version>
+    <description>
+Splunk OpenTelemetry Connector is a distribution of the OpenTelemetry Collector. It provides a unified way to receive, process, and export metric, trace, and log data for Splunk Observability Cloud.
+
+The splunk-otel-collector looks for a configuration file at `\ProgramData\Splunk\OpenTelemetry Collector\agent_config.yaml`.  If this file does not already exist during installation, a default config file will be copied into place by the installer.
+
+#### Package Parameters
+The following package parameters are available:
+
+ * `/SPLUNK_ACCESS_TOKEN` - The splunk access token (org token) used to send metric data to OTEL. If not specified default is to `1234`

```

Comment:
There should be no default token.  If not specified, do not start the collector service after installation.

Reply:
Same changes are made.